src/controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,112 @@
+import { Request, Response } from 'express';
+import { UserRepository } from '../../repositories';
+
+const userRepository = new UserRepository();
+
+class TraineeRoutes {
+// Function to fetch all trainees
+    public async get(request: Request, response: Response, next) {
+        try {
+            const {skip , limit} = request.query;
+            const count = await userRepository.count();
```

Comment:
Why are not applying condition of role == trainee over here as it should only return count of trainee.

Reply:
role is being checked during implementation of count method in versionableRepository